src/files/FilesPage.js

```diff
@@ -231,11 +232,13 @@ class FilesPage extends React.Component {
 
               { files.type === 'directory'
                 ? <div className='ml-auto flex items-center'>
-                  <span
-                    className='mr3 flex items-center f6 gray pointer'
+                  <Button
+                    className='mr1 flex items-center f6 fw3 pointer'
+                    color='charcoal-muted'
+                    bg='bg-transparent'
                     onClick={() => this.showNewFolderModal()}>
-                    <FolderIcon className='mr1 fill-gray w2 o-80 glow' />{t('newFolder')}
-                  </span>
+                    <FolderIcon viewBox='10 15 80 80' height='20px' className='fill-charcoal-muted w2' />{t('newFolder')}
```

Comment:
@fsdiogo is there a problem with the positioning of this icon that we should fix in the svg?

Reply:
I don't think so, I just tweaked it a bit so it didn't make its parent container grow!